transport/tchannel/handler.go

```diff
@@ -116,12 +116,16 @@ func (h handler) handle(ctx context.Context, call inboundCall) {
 	responseWriter := h.newResponseWriter(call.Response(), call.Format(), h.headerCase)
 	defer responseWriter.ReleaseBuffer()
 
+	span := opentracing.SpanFromContext(ctx)
```

Comment:
The span here is not the right tchannel inbound span.
The tchannel inbound span is available after `tchannel.ExtractInboundSpan(...)` is called in `callHandler` method, and you can get the span from the context.